client/my-sites/theme/controller.jsx

```diff
@@ -29,12 +28,10 @@ let themeDetailsCache = new Map();
 
 export function makeElement( ThemesComponent, Head, store, props ) {
 	return(
-		<ReduxProvider store={ store }>
-			<Head title={ props.title } description={ props.description } type={ 'website' }
-				canonicalUrl={ props.canonicalUrl } image={ props.image } tier={ props.tier || 'all' }>
-				<ThemesComponent { ...omit( props, [ 'title' ] ) } />
-			</Head>
-		</ReduxProvider>
+		<Head title={ props.title } description={ props.description } type={ 'website' }
+			canonicalUrl={ props.canonicalUrl } image={ props.image } tier={ props.tier || 'all' }>
+			<ThemesComponent { ...omit( props, [ 'title' ] ) } />
```

Comment:
> It's strange that we pass everything but `title` to me. Why not `title`? Why not skip `description` and `image` etc…

I'm not familiar with the history, perhaps @ockham knows? I'd probably side with you on the point of being explicit, though.